logging/logging/__init__.py

```diff
@@ -152,12 +152,9 @@ def close(self):
 class Formatter:
 
     converter = utime.localtime
-    default_time_format = '%Y-%m-%d %H:%M:%S'
-    default_msec_format = '%s,%03d'
 
-    def __init__(self, fmt=None, datefmt=None, style="%"):
```

Comment:
Well, please don't remove it here, likewise pass it down like you do. Someone can subclass this class and implement full format.

Reply:
OK, I wasn't so sure which one is the better solution...removing a non-used argument or leaving it there and forcing it to be None. Personally I tend to not implementing any unused arguments to keep the codebase cleaner and simpler. But I'm OK with passing them on as you suggested.